src/classes/book/book.rb

```diff
@@ -13,8 +13,34 @@ def initialize(publisher, cover_state, *args)
   def can_be_archived?
     (Date.today - @publish_date) > 3652 or @cover_state == 'bad'
   end
+
+  def self.add_book
+    puts 'ADD A NEW BOOK'
+    print 'Enter publisher: '
+    publisher = gets.chomp.to_s
+    print 'Enter cover state (good/bad):'
+    cover_state = gets.chomp.to_s
+    print 'Enter publish date (YYYY-MM-DD):'
+    date = gets.chomp.to_s
+    print 'Enter genre:'
+    genre = gets.chomp.to_s
+    print 'Enter author:'
+    author = gets.chomp.to_s
+    print 'Enter label:'
+    label = gets.chomp.to_s
```

Comment:
Kindly add a method to ask the user for label parameters (title and color) instead of label.

Reply:
Aaaah ok!! thanks!